docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1599,25 +1599,33 @@ The `ActionProfile` message includes the following fields:
   the `selector_size_semantics` value as described below.
 
 * `max_group_size`, an `int32` which is 0 for an Action Profile, or, for an
-  Action Selector, its semantics is specified by the
-  `selector_size_semantics` value as described below.
+  Action Selector, its semantics is specified by the  `selector_size_semantics`
+  value as described below.
   The `max_group_size` must be no larger than `size`. PSA programs can use the
   `@max_group_size` annotation to provide this value for Action Selectors.
   If the annotation is omitted, the P4Info field will default to 0.
 
-* `selector_size_semantics`, an enum for Action Selectors that
+* `selector_size_semantics`, a oneof for Action Selectors that
   specifies how `size` and `max_group_size` are interpreted. It can be either:
-    * `SUM_OF_WEIGHTS`, indicating that `size` and `max_group_size` represent
+    * `sum_of_weights`, indicating that `size` and `max_group_size` represent
       the maximum sum of weights that can be present across all selector groups
       and within a single selector group respectively.
-    * `SUM_OF_MEMBERS`, indicating that `size` and `max_group_size` represent
+    * `sum_of_members`, indicating that `size` and `max_group_size` represent
       the maximum number of members that can be present across all selector
-      groups and within a single selector group respectively.
+      groups and within a single selector group respectively. The `SumOfMembers`
+      message used to represent this value also contains an optional int32
+      `max_member_weight`, which indicates the maximum weight of each
+      individual member. If unset, any 32-bit integer is allowable for weight.
```

Comment:
I'd say "allowed" sounds better than "allowable" here, but that may be a matter of preference